calcite-qml/Calcite/Calcite.qml

```diff
@@ -43,18 +43,28 @@ QtObject {
 
     readonly property color background: theme === Calcite.Theme.Light ? offWhite : "#353535"
 
+    readonly property color background2: theme === Calcite.Theme.Light ? "#C7EAff" : "#151515"
+
+    readonly property color background3: theme === Calcite.Theme.Light ? "FFFFFF" : "#404040"
+
     readonly property color foreground1: theme === Calcite.Theme.Light ? "#FFFFFF" : "#2B2B2B"
 
     readonly property color foreground2: theme === Calcite.Theme.Light ? "#F3F3F3" : "#202020"
 
     readonly property color foreground3: theme === Calcite.Theme.Light ? "#EAEAEA" : "#151515"
 
+    readonly property color foreground4: theme === Calcite.Theme.Light ? "#004874" : "#59d6ff"
+
+    readonly property color foreground5: theme === Calcite.Theme.Light ? "#EAEAEA" : "#2B2B2B"
+
     readonly property color text1: theme === Calcite.Theme.Light ? "#151515" : "#FFFFFF"
 
     readonly property color text2: theme === Calcite.Theme.Light ? "#4A4A4A" : "#BFBFBF"
 
     readonly property color text3: theme === Calcite.Theme.Light ? "#6A6A6A" : "#9F9F9F"
 
+    readonly property color text4: theme === Calcite.Theme.Light ? "#2B2B2B" : "#FFFFFF"
+
```

Comment:
🔧  We had a chat on restricting this list of var to: `https://github.com/Esri/calcite-colors/blob/master/src/colors.ts#L832`.

I am not against updating the existing color profiles to be conformant with figma/react.  But it is worth noting that as it currently stands,  that the ToolButtons look different in terms of highlighting behavior and text color to the rest of the available controls.

<img width="495" alt="image" src="https://user-images.githubusercontent.com/42203545/152181151-56448380-8c28-4766-bbac-61a85d3510aa.png">

Looping in @nCastle1 as he might have an idea of what the most "correct" color scheme is.